packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -6,6 +6,41 @@ import 'package:meta/meta.dart';
 
 import 'print.dart';
 
+/// The various priority levels used to filter which diagnostics are shown and
+/// omitted.
+///
+/// Trees of Flutter diagnostics can be very large so filtering the Diagnostics
+/// shown matters. Typically filtering to only show diagnostics with at least
+/// level debug is appropriate.
+enum DiagnosticLevel {
+  /// Use this level for diagnostic properties that match their default value
```

Comment:
Add a paragraph above the current paragraph for `fine` that describes this value, e.g. "Trivial or redundant diagnostics information."

Reply:
done